src/main/java/com/parsing/parsers/pdf/parsing/ManyTableDetector.java

```diff
@@ -61,12 +61,10 @@ private int groupVerticalLines(List<double[]> lines, List<Double> mergedY) {
         for (Double yUp : yUpList) {
             for (Double yDown : yDownList) {
                 if (yDown < yUp) {
-                    log.debug("Ydown " + yDown + " Yup" + yUp);
```

Comment:
I think that we need to consider using debug log level for this type of exception. And we can use INFO for logs that help us to follow the program flow. Do you need it at all?